package.json

```diff
@@ -109,7 +108,8 @@
     "rollup-plugin-replace": "^2.1.0",
     "rollup-plugin-terser": "^7.0.2",
     "sinon": "^9.2.0",
-    "typescript": "^4.0.3"
+    "typescript": "^4.0.3",
+    "uuid": "^8.3.1"
```

Comment:
any plan to use it in dev too ?

Reply:
too many file changes which clash with other PRs, another time.